config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -582,10 +582,12 @@ presubmits:
       containers:
         - image: gcr.io/k8s-testimages/kubekins-e2e:v20210615-c973edd-master
           args:
-            - --repo=k8s.io/kubernetes=$(PULL_REFS)
-            - --repo=k8s.io/release
-            - --timeout=240
             - --root=/go/src
+            - "--job=$(JOB_NAME)"
+            - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+            - "--service-account=/etc/service-account/service-account.json"
+            - "--upload=gs://kubernetes-jenkins/pr-logs"
+            - "--timeout=440"
```

Comment:
This is a very long timeout, does it need to be 440 (7h 20m) or can we leave it at 240 (4h)? The Fedora job took under 2h to run iirc.

Reply:
Yeah, I used a longer timeout since Ubuntu need a reboot to enable swap accounting, but I guess 440 is probably too much buffer. Will update to 2h as suggested.